content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,231 @@
+    the webhook injected a sidecar into the example pod:
+
+    {{< text plain >}}
+    NAME    READY   STATUS    RESTARTS   AGE
+    nginx-app   2/2     Running   0          10s
+    {{< /text >}}
+
+1.  Check that the validation webhook is working:
+
+    {{< text bash >}}
+    $ kubectl create namespace test-validation
+    $ kubectl apply -n test-validation -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: invalid-gateway
+    spec:
+      selector:
+        # DO NOT CHANGE THESE LABELS
+        # The ingressgateway is defined in install/kubernetes/helm/istio/values.yaml
+        # with these labels
+        istio: ingressgateway
+    EOF
+    {{< /text >}}
+
+    The output from the gateway creation command should show the following output. The error
+    in the output indicates that the validation webhook checked the gateway's configuration YAML file:
+
+    {{< text plain >}}
+    Error from server: error when creating "invalid-gateway.yaml": admission webhook "pilot.validation.istio.io" denied the request: configuration is invalid: gateway must have at least one server
+    {{< /text >}}
+
+## Show webhook configurations
+
+1.  Show the configuration of the webhooks:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status
+    {{< /text >}}
+
+1.  If you named the sidecar injector's configuration `istio-sidecar-injector`,
+use the following command to show the configuration of the sidecar injector:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  If you named Galley's configuration `istio-galley`, show Galley's configuration with the following command:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook status --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## Disable webhook configurations
+
+1.  Disable Galley and the sidecar injector when using their default webhook configuration names:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable
+    {{< /text >}}
+
+1.  If you named the sidecar injector's configuration `istio-sidecar-injector`,
+disable the webhook with the following command:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --validation=false --injection-config=istio-sidecar-injector
+    {{< /text >}}
+
+1.  If you name Galleys's configuration `istio-galley`, disable the webhook with the following command:
+
+    {{< text bash >}}
+    $ istioctl experimental post-install webhook disable --injection=false --validation-config=istio-galley
+    {{< /text >}}
+
+## Cleanup
+
+After completing this tutorial, you may delete the testing cluster created
+at the beginning of this tutorial. You may also run the following command to delete
+the resources created.
```

Comment:
~After completing this tutorial, you may delete the testing cluster created
at the beginning of this tutorial.~ You ~may also~ can run the following command to delete
the resources created at the beginning of this tutorial.

Reply:
Revised accordingly.